api-reference/authentication/v2/authentication.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Authentication"
+api: "POST https://travtus-api.auth.us-east-2.amazoncognito.com/oauth2/token"
+version: "v2"
+---
+
+Travtus API uses token authentication. All API calls must include the following header:
+
+```bash
+'Authorization': 'Bearer <team_token>'
+```
+
+### Query Parameters
+
+<ParamField query="client_id" type="string">
+  Your API client_id.
+</ParamField>
+
+<ParamField query="client_secret" type="string">
+  Your API client_secret.
+</ParamField>
+
+<ParamField query="grant_type" type="string">
+    Must be `client_credentials`.
+</ParamField>
+
+<ParamField query="scope" type="string">
+    The API scopes granted to the access token. Please use `travtus-api/knowledge-graph`.
```

Comment:
Is this for external users? I.e., our customers?

Reply:
Yes. When a client requests a token, they must tell the resource server the scope they need. When we generate an API client credential, we associate custom scopes with the API client. Therefore, they can only request scopes that are associated.

Reply:
Why do they need to request scopes, we can't just provide them all of the scopes available?

My concern is that this scope is something internal to our system and not something they should necessarily need to know about.

Reply:
I'm speaking largely about the developer experience for them, not because that scope is confidential or anything like that.